client/keys.go

```diff
@@ -14,7 +14,7 @@
 
 package client
 
-//go:generate codecgen -r "Node|Response|Nodes" -o keys.generated.go keys.go
+//go:generate codecgen -d 1819 -r "Node|Response|Nodes" -o keys.generated.go keys.go
```

Comment:
If you don't set it explicitly - the it is random based on the current time, which makes the generated code differ every time it's generated. I think having it always be the same is expected.